### PR TITLE
Add Milk Truck keyboard

### PR DESCRIPTION
**Pull request by rjboone**
## Description

This pull request contains the necessary files to add Milk Truck to QMK main.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Initial addition of keyboard

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/milk_truck/config.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x0001
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0001
```
Comment (drashna) [inline keyboards/milk_truck/config.h:23]: Please chaleg

File: keyboards/milk_truck/config.h [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define RGB_DI_PIN F0
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/milk_truck/config.h:59]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,152 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/milk_truck/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+    [_FKEY_RGB] = LAYOUT(
+      RGB_TOG, RGB_RMOD, RGB_MOD,          KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+      RGB_HUI, RGB_SAI,  RGB_VAI,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+      RGB_HUD, RGB_SAD,  RGB_VAD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,      KC_TRNS,        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS
+    )
+};
+
+#ifdef OLED_DRIVER_ENABLE  //Special thanks to Sickbabies for this great OLED widget!
```
Comment (drashna) [inline keyboards/milk_truck/keymaps/default/keymap.c:39]:
```suggestion
#ifdef OLED_ENABLE  //Special thanks to Sickbabies for this great OLED widget!
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+void render_mod_status(uint8_t modifiers) {
+       oled_write_P(PSTR("-----"), false);
+    oled_write_ln_P(PSTR("SHFT"), (modifiers & MOD_MASK_SHIFT));
+    oled_write_ln_P(PSTR("ALT"), (modifiers & MOD_MASK_ALT));
+    oled_write_ln_P(PSTR("CTRL"), (modifiers & MOD_MASK_CTRL));
+    oled_write_ln_P(PSTR("GUI"), (modifiers & MOD_MASK_GUI));
+}
+
+void oled_task_user(void) {
```
Comment (drashna) [inline keyboards/milk_truck/keymaps/default/keymap.c:137]:
```suggestion
bool oled_task_user(void) {
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+    render_milktruck_logo();
+    oled_set_cursor(0,3);
+    render_layer_status();	// Renders the current keyboard state (layer, lock, caps, scroll, etc)
+	render_mod_status(get_mods()|get_oneshot_mods());
+	render_keylock_status(host_keyboard_led_state());
+	render_keylogger_status();
+}
```
Comment (drashna) [inline keyboards/milk_truck/keymaps/default/keymap.c:144]:
```suggestion

    return false;
}
```

File: keyboards/milk_truck/keymaps/rjboone/keymap.c [hunk 7]
```diff
@@ -0,0 +1,153 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/milk_truck/keymaps/rjboone/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/milk_truck/readme.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+![Milk Truck](https://imgur.com/NUmiTDS.jpg)
+
+A dairy-filled, left-handed, numpad-having, extended-minivan, forty-percent keyboard.
+
+* Keyboard Maintainer: [Ryan Boone](https://github.com/rjboone) (Drmmr#0004)
+* Hardware Supported: Milk Truck
+* Hardware Availability: Group buy from [P3D Store](https://p3dstore.com)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/milk_truck/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/milk_truck/rules.mk [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+OLED_DRIVER_ENABLE = yes
```
Comment (drashna) [inline keyboards/milk_truck/rules.mk:19]:
```suggestion
OLED_ENABLE = yes
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/milk_truck/readme.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
+# Milk Truck
+
+![Milk Truck](https://imgur.com/NUmiTDS.jpg)
```
Comment (fauxpark) [inline keyboards/milk_truck/readme.md:3]:
```suggestion
![Milk Truck](https://imgur.com/NUmiTDSh.jpg)
```

File: keyboards/milk_truck/keymaps/rjboone/readme.md [hunk 11]
```diff
@@ -0,0 +1 @@
+# rjboone's Milk Truck Layout
```
Comment (fauxpark) [inline keyboards/milk_truck/keymaps/rjboone/readme.md:1]: `rules.mk` here can be deleted.

File: keyboards/milk_truck/keymaps/default/readme.md [hunk 12]
```diff
@@ -0,0 +1 @@
+# Default Milk Truck Layout
```
Comment (fauxpark) [inline keyboards/milk_truck/keymaps/default/readme.md:1]: `rules.mk` here can be deleted.

File: keyboards/milk_truck/info.json [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+}
```
Comment (fauxpark) [inline keyboards/milk_truck/info.json:6]:
```suggestion
{
    "keyboard_name": "milk_truck",
    "url": "",
    "maintainer": "rjboone",
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},

                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6, "y": 0},
                {"x": 7, "y": 0},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0},
                {"x": 14, "y": 0},
                {"x": 15, "y": 0, "w": 1.75},

                {"x": 0, "y": 1},
                {"x": 1, "y": 1},
                {"x": 2, "y": 1},

                {"x": 4, "y": 1, "w": 1.25},
                {"x": 5.25, "y": 1},
                {"x": 6.25, "y": 1},
                {"x": 7.25, "y": 1},
                {"x": 8.25, "y": 1},
                {"x": 9.25, "y": 1},
                {"x": 10.25, "y": 1},
                {"x": 11.25, "y": 1},
                {"x": 12.25, "y": 1},
                {"x": 13.25, "y": 1},
                {"x": 14.25, "y": 1},
                {"x": 15.25, "y": 1, "w": 1.5},

                {"x": 0, "y": 2},
                {"x": 1, "y": 2},
                {"x": 2, "y": 2},
                {"x": 3, "y": 2},

                {"x": 4, "y": 2, "w": 1.75},
                {"x": 5.75, "y": 2},
                {"x": 6.75, "y": 2},
                {"x": 7.75, "y": 2},
                {"x": 8.75, "y": 2},
                {"x": 9.75, "y": 2},
                {"x": 10.75, "y": 2},
                {"x": 11.75, "y": 2},
                {"x": 12.75, "y": 2},
                {"x": 13.75, "y": 2},
                {"x": 14.75, "y": 2},
                {"x": 15.75, "y": 2},

                {"x": 0, "y": 3},
                {"x": 1, "y": 3},
                {"x": 2, "y": 3},
                {"x": 3, "y": 3},

                {"x": 4, "y": 3},
                {"x": 5.5, "y": 3, "w": 1.5},
                {"x": 7, "y": 3},
                {"x": 8, "y": 3, "w": 2.25},
                {"x": 10.25, "y": 3, "w": 2},
                {"x": 12.25, "y": 3},
                {"x": 13.25, "y": 3, "w": 1.25},
                {"x": 14.5, "y": 3},
                {"x": 15.5, "y": 3, "w": 1.25}
            ]
        }
    }
}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/milk_truck/readme.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (the top left key) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB
+* **Keycode in layout**: Press the key mapped to `RESET`
```
Comment (noroadsleft) [inline keyboards/milk_truck/readme.md:27]:
```suggestion
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (the top left key) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB
* **Keycode in layout**: Press the key mapped to `RESET`

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/milk_truck/milk_truck.h [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT( \
```
Comment (noroadsleft) [inline keyboards/milk_truck/milk_truck.h:22]: As the PCB supports multiple layouts, this should be named:

```suggestion
#define LAYOUT_all( \
```

File: keyboards/milk_truck/keymaps/rjboone/keymap.c [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+
+enum layers{
+  _BASE,
+  _NUM_SYM,
+  _FKEY_RGB,
+  _NAV,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/rjboone/keymap.c:26]:
```suggestion
    [_BASE] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/rjboone/keymap.c [hunk 17]
```diff
@@ -0,0 +1,170 @@
...
+      KC_TRNS,   KC_TRNS,   KC_TRNS,            KC_ESC,    KC_Q,    KC_W,    KC_E,    KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,     KC_O,    KC_P,    KC_BSPC,
+      KC_TRNS,   KC_TRNS,   KC_TRNS,            KC_TAB,    KC_A,    KC_S,    KC_D,    KC_F,  KC_G,  KC_H,  KC_J,  KC_K,     KC_L,    KC_SCLN, KC_QUOT,
+      KC_MPRV,   KC_MPLY,   KC_MNXT,  KC_TRNS,  KC_LSFT,   KC_Z,    KC_X,    KC_C,    KC_V,  KC_B,  KC_N,  KC_M,  KC_COMM,  KC_DOT,  KC_SLSH, KC_RSFT,
+      KC_VOLD,   KC_TRNS,   KC_VOLU,  KC_TRNS,  KC_LCTRL,  KC_LALT, KC_LGUI, LT(_NUM_SYM, KC_ENT), KC_SPC, MO(_NAV), KC_RALT, MO(_FKEY_RGB), KC_RCTL
+    ),
+
+    [_NUM_SYM] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/rjboone/keymap.c:33]:
```suggestion
    [_NUM_SYM] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/rjboone/keymap.c [hunk 18]
```diff
@@ -0,0 +1,170 @@
...
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MINS, KC_LBRC, KC_RBRC,  KC_EQL, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BSLS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,      KC_RSFT,        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [_NAV] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/rjboone/keymap.c:40]:
```suggestion
    [_NAV] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/rjboone/keymap.c [hunk 19]
```diff
@@ -0,0 +1,170 @@
...
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_UP,    KC_TRNS,    KC_TRNS,    KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_LEFT, KC_DOWN, KC_RIGHT, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,      KC_RSFT,        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [_FKEY_RGB] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/rjboone/keymap.c:47]:
```suggestion
    [_FKEY_RGB] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 20]
```diff
@@ -0,0 +1,170 @@
...
+
+enum layers{
+  _BASE,
+  _NUM_SYM,
+  _FKEY_RGB,
+  _NAV,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/default/keymap.c:26]:
```suggestion
    [_BASE] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 21]
```diff
@@ -0,0 +1,170 @@
...
+      KC_P7,   KC_P8,   KC_P9,            KC_ESC,    KC_Q,    KC_W,    KC_E,    KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,     KC_O,    KC_P,    KC_BSPC,
+      KC_P4,   KC_P5,   KC_P6,            KC_TAB,    KC_A,    KC_S,    KC_D,    KC_F,  KC_G,  KC_H,  KC_J,  KC_K,     KC_L,    KC_SCLN, KC_ENT,
+      KC_P1,   KC_P2,   KC_P3,   KC_UP,   KC_LSFT,   KC_Z,    KC_X,    KC_C,    KC_V,  KC_B,  KC_N,  KC_M,  KC_COMM,  KC_DOT,  KC_SLSH, MO(_FKEY_RGB),
+      KC_P0,   KC_PENT, KC_LEFT, KC_DOWN, KC_RIGHT,  KC_LCTL, KC_LALT,    MO(_NUM_SYM),      KC_SPC,        KC_RALT,  KC_RGUI, KC_RCTL, KC_BSLS
+    ),
+
+    [_NUM_SYM] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/default/keymap.c:33]:
```suggestion
    [_NUM_SYM] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 22]
```diff
@@ -0,0 +1,170 @@
...
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LBRC, KC_MINS, KC_EQL,  KC_RBRC, KC_TRNS, KC_TRNS, KC_TRNS, KC_QUOT,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,      KC_TRNS,        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [_NAV] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/default/keymap.c:40]:
```suggestion
    [_NAV] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/default/keymap.c [hunk 23]
```diff
@@ -0,0 +1,170 @@
...
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_UP,    KC_TRNS,    KC_TRNS,    KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_LEFT, KC_DOWN, KC_RIGHT, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,      KC_RSFT,        KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+
+    [_FKEY_RGB] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/default/keymap.c:47]:
```suggestion
    [_FKEY_RGB] = LAYOUT_all(
```

File: keyboards/milk_truck/keymaps/default/config.h [hunk 24]
```diff
@@ -0,0 +1,16 @@
+/* Copyright 2022 rjboone
```
Comment (noroadsleft) [inline keyboards/milk_truck/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/milk_truck/info.json [hunk 25]
```diff
@@ -0,0 +1,77 @@
+{
+    "keyboard_name": "milk_truck",
+    "url": "",
+    "maintainer": "rjboone",
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/milk_truck/info.json:6]:
```suggestion
        "LAYOUT_all": {
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/15651/files).

**Pull request closed without merging**